receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -112,15 +115,28 @@ const (
 
 // New creates a TraceReceiver that receives traffic as a collector with both Thrift and HTTP transports.
 func New(ctx context.Context, config *Configuration, nextConsumer consumer.TraceConsumer, logger *zap.Logger) (receiver.TraceReceiver, error) {
-	return &jReceiver{
+	jR := &jReceiver{
 		config:          config,
 		defaultAgentCtx: observability.ContextWithReceiverName(context.Background(), "jaeger-agent"),
 		nextConsumer:    nextConsumer,
 		tchanServer: &jTchannelReceiver{
 			nextConsumer: nextConsumer,
 		},
 		logger: logger,
-	}, nil
+	}
+
+	if config.RemoteSamplingEndpoint != "" {
```

Comment:
Should this block be in `StartTraceReception`?

Reply:
The `config` object is passed down to the `New()` initialiser, would be difficult to implement in `StartTraceReception `